websites/R/Radiko/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Munn",
+    "id": "379616881272946688"
+  },
+  "contributors": [
+    {
+      "name": "Krauchelli",
+      "id": "229334514084741120"
+    }
+  ],
+  "service": "Radiko",
+  "description": {
+    "en": "Radiko is Japanese website that provide radio livestream in real time.",
+    "ja_JP": "ラジコは、あなたが今いるエリアで放送しているラジオ局をパソコンやスマートフォンで聴くことのできるサービスです。"
+  },
+  "url": "radiko.jp",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/ig69txc.png",
```

Comment:
Here I removed the white edges for you from the logo: https://i.imgur.com/2jmN74S.png

Reply:
Wait, i'll change